plugins/terafast-bulk-loader/impl/src/test/java/org/pentaho/di/trans/steps/terafastbulkloader/TeraFastTest.java

```diff
@@ -19,7 +19,7 @@
  * limitations under the License.
  *
  ******************************************************************************/
```

Comment:
Need to update the copyright date in this file.

Reply:
Done